### ISSUE TITLE
How to download nightbuild x360ce ?

### ISSUE THREAD

**Issue by Karaulov** [x360ce/x360ce#752]
https://ci.appveyor.com/x360ce/x360ce - is empty, where is located nightbuilds x360ce ?

**Comment by jamesst20** [comment]
https://github.com/x360ce/x360ce/blob/master/Wiki/BetaTesting.md

**Comment by Karaulov** [comment]
No, this beta version, 26 day old (https://i.imgur.com/f1kfZhh.png) i need download latest nightbuilds (autobuilds) 

why https://ci.appveyor.com/project/x360ce/x360ce not works ? "Project not found or access denied."
